site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-mac.rst

```diff
@@ -54,14 +104,42 @@ Make sure you have first set up :ref:`LAN access<lan-mac>`.  Then do the followi
 
 1. Download the appropriate desktop client from https://nextcloud.com/install/#install-clients
 2. Open the client and click "Log In"
+
+  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step2.png
+    :width: 50%
+    :alt: nextcloud-login
+
 3. From your server's Nextcloud Service page, go to "Interfaces" and copy the LAN address
+
+  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step3.png
+    :width: 50%
+    :alt: nextcloud-login
+
 4. Enter your LAN address under "Server Address" and click "Next"
-5. You will be asked to Trust your server's certificate, which is safe to do as you generate and sign this during LAN Setup
-6. Tick the box for "Trust this certificate anyway" and click "Next"
-7. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards
-8. Next, configure the local directory that you want to sync with Nextcloud. You may use the default or change it, and edit the sync settings to desired. When satisfied, click "Connect"
-9. Files will begin to sync immediately and you will see a green check when this is complete.
-10. That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages
+
+  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step4.png
+    :width: 50%
+    :alt: nextcloud-login
+
+5. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards
```

Comment:
```suggestion
5. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards.
```